__tests__/app.test.js

```diff
@@ -0,0 +1,29 @@
+const request = require('supertest');
+const app = require('../app')
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+const data = require('../db/data/test-data/index');
+
+
+beforeEach(()=>seed(data));
+afterAll(()=>db.end());
+
+describe('/api/topics', () => {
+    test('200: get all topics', () => {
+      return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.topics).toBeInstanceOf(Array);
+          expect(body.topics[0]).toMatchObject({slug:expect.any(String), description:expect.any(String)});
```

Comment:
Great test. As it's gonna reseed before each test, so the array won't change, thus, it would be better to check the length of the array as well. And we want every object is following the same shape, so a forEach would be better to check that instead of only check the first object